sdks/python/apache_beam/internal/auth.py

```diff
@@ -82,6 +82,9 @@ def __init__(self, user_agent=None):
         None,  # token_uri
         user_agent)
 
+  @retry.with_exponential_backoff(
+      num_retries=3, initial_delay_secs=1.0,
```

Comment:
I removed the overrides for num_retries and initial_delay_secs. This should use defaults.

As a side note maybe we should update the defaults. Currently they are num_retries=16, initial_delay_secs=5.0. That would cause a 90+hrs sleep before the final retry.